ui/src/components/schemas/ImportSchemaPreview.tsx

```diff
@@ -29,7 +29,9 @@ export function ImportSchemaPreview({
   const bigintResult = getBigint(jsonLdType);
   const bigint = bigintResult && bigintResult.success ? bigintResult.data : null;
   const schemaHashResult = getSchemaHash(jsonLdType);
-  const schemaHash = schemaHashResult && schemaHashResult.success ? schemaHashResult.data : null;
+  const schemaHash = schemaHashResult.success
+    ? schemaHashResult.data
+    : schemaHashResult.error.message;
```

Comment:
This is no place for an error. Since this error is not a blocker, we better show a toast and default to `"-"`